packages/expo-notifications/README.md

```diff
@@ -1 +1,367 @@
+- [`requestPermissionsAsync`](#requestpermissionsasync) -- requests permissions related to notifications
+
+# Installation in managed Expo projects
+
+This library is not yet usable within managed projects &mdash; it is likely to be included in an upcoming Expo SDK release.
+
+# Installation in bare React Native projects
+
+For bare React Native projects, you must ensure that you have [installed and configured the `react-native-unimodules` package](https://github.com/unimodules/react-native-unimodules) before continuing.
+
+### Add the package to your npm dependencies
+
+```
+expo install expo-notifications
+```
+
+### Configure for iOS
+
+Run `pod install` in the `ios` directory after installing the npm package.
+
+### Configure for Android
+
+Ensure that your project is configured for Firebase (if it is, it should contain a `google-services.json` file in `android/app`).
+
+# Contributing
+
+Contributions are very welcome! Please refer to guidelines described in the [contributing guide](https://github.com/expo/expo#contributing).
+
+------
+
+### `getExpoPushTokenAsync(options: ExpoTokenOptions): ExpoPushToken`
+
+Returns an Expo token that can be used to send a push notification to this device using Expo push notifications service. [Read more in the Push Notifications guide](https://docs.expo.io/versions/latest/guides/push-notifications/).
+
+> **Note:** For Expo backend to be able to send notifications to your app, you will need to provide it with push notification keys. This can be done using `expo-cli` (`expo credentials:manager`). [Read more in the TODO guide](#todo).
+
+#### Arguments
+
+This function accepts an optional object allowing you to pass in configuration, consisting of fields (all are optional, but some may have to be defined if configuration cannot be inferred):
+
+- **experienceId (_string_)** -- The ID of the experience to which the token should be attributed. Defaults to [`Constants.manifest.id`](https://docs.expo.io/versions/latest/sdk/constants/#constantsmanifest) exposed by `expo-constants`. You may need to define it in bare workflow, where `expo-constants` doesn't expose the manifest.
+- **devicePushToken ([_DevicePushToken_](#devicepushtoken))** -- The device push token with which to register at the backend. Defaults to a token fetched with [`getDevicePushTokenAsync()`](#getdevicepushtokenasync).
+- **appId (_string_)** -- The ID of the application to which the token should be attributed. Defaults to [`Application.applicationId`](https://docs.expo.io/versions/latest/sdk/application/#applicationapplicationid) exposed by `expo-application`.
+- **deviceId (_string_)** -- The ID of the application installation. `expo-notifications` is capable of generating one for you and it defaults to it. Most probably you won't need to customize that.
+- **type (_string_)** -- Type of token sent to the server. Inferred from `devicePushToken`. Most probably you won't need to customize that.
```

Comment:
I think if we're documenting these it's still worth documenting the effect of setting it / when you might want to set it explicitly.